### ISSUE TITLE
[FEATURE] Russian translation

### ISSUE THREAD

**Issue by rezchikov** [tiangolo/fastapi#1362]
### 🇷🇺 Russian translation

Comrades from Russia need this :-)
Let me know if you need a more precise plan for translating documentation into Russian. I created this issue to track and coordinate translation activities.

_(in Soviet Russia documentation translates YOU)_

**Comment by fojetin** [comment]
See my review please https://github.com/tiangolo/fastapi/pull/1363#pullrequestreview-421383447

**Comment by tiangolo** [comment]
Awesome!

> (in Soviet Russia documentation translates YOU)

This made me laugh :joy: 

---

I just made a "meta-issue" with translations to pin that one and refer to each issue coordinating translations (including this one) from it.

**Comment by fojetin** [comment]
Hello, comrade @tiangolo !
Unfortunately comrade @rezchikov hasn't been online for a month. I hope he’s not in gulag 😢 

I want to develop Russian translation, how is it better to do this? Create a new pull request?

**Comment by tiangolo** [comment]
Hey @fojetin !

I just added and merged a small PR with the basic setup for Russian translations. :heavy_check_mark: 

That way you (and anyone else that wants to help) can take one page and start right away.

The `index.md` is still not translated, I guess we can wait a bit for @rezchikov's PR.

But having the basic setup should allow you to take any other page and translate it :rocket: 

Thanks for the help! :bow:  :tada:

**Comment by SirTelemak** [comment]
@tiangolo I made my review for @rezchikov PR and want to contribute for FastAPI too, starting with translation as boost start for now :) 
@fojetin we can split documentation for now so we won't translate same things simultaneously

**Comment by ycd** [comment]
Looks like the creator of this issue is gone, assuming the first PR became cannot be controllable anymore, if anyone willing to continue on this, please start with a brand new PR.

**Comment by Kludex** [comment]
Yeah, actually this is blocking the translations here... #1363 is actually the setup + index.md, if someone is interested on contributing for Russian translations, please create first the setup then you can work the `index.md` on another PR.

Well noticed @ycd :sunglasses:

**Comment by Xewus** [comment]
https://github.com/tiangolo/fastapi/pull/5577
:globe_with_meridians: Add Russian translation for `docs/ru/docs/fastapi-people.md`.

**Comment by Xewus** [comment]
[#5665](https://github.com/tiangolo/fastapi/pull/5665)
:globe_with_meridians: Add Russian translation for docs/ru/docs/help-fastapi.md
